rest/src/main/kotlin/Image.kt

```diff
@@ -22,11 +20,12 @@ public class Image private constructor(public val data: ByteArray, public val fo
         }
 
         public suspend fun fromUrl(client: HttpClient, url: String): Image = with(Dispatchers.IO) {
-            val call = client.request<HttpResponse>(url) { method = HttpMethod.Get }
+            val call = client.get(url)
             val contentType = call.headers["Content-Type"]
                 ?: error("expected 'Content-Type' header in image request")
 
-            val bytes = call.content.toByteArray()
+            @Suppress("EXPERIMENTAL_API_USAGE")
+            val bytes = call.body<ByteArray>()
```

Comment:
```suggestion
            val bytes = call.body<ByteArray>()
```
not necessary, only the `content` property used previously became internal api in ktor 2